languages/ruby/spec/oso/polar/data_filtering/parity_spec.rb

```diff
@@ -258,7 +258,7 @@
 
     it 'test_specializers' do
       subject.load_str <<~POL
-        allow(foo: Foo,             "NoneNone", log) if foo = log.foo;
+        allow(foo,                  "NoneNone", log) if foo = log.foo;
```

Comment:
Why? Doesn't this change what's being tested?

Reply:
This is how it should be. `NoneNone` means no specializer on either side.

Reply:
ohh ok